frontend/src/entities/dataset/dataset.service.tsx

```diff
@@ -150,12 +156,19 @@ export async function uploadResources (datasetContext: DatasetContext, resourceO
                     `Successfully uploaded ${successCount}/${resourceObjects.length} file(s).`,
                     'in-progress',
                     'upload-notification',
-                    (<ProgressBar
-                        label='Uploading files to dataset'
-                        description={`Uploading ${resourceObject.name}`}
-                        value={successCount / resourceObjects.length * 100}
-                        variant='flash'
-                    />),
+                    (<ColumnLayout columns={2}>
+                        <ProgressBar
+                            label='Uploading files to dataset'
+                            description={`Uploading ${resourceObject.name}`}
+                            value={successCount / resourceObjects.length * 100}
+                            variant='flash'
+                        />
+                        <Box float='right'>
+                            <Button onClick={() => breakout = true}>Stop</Button>
```

Comment:
Can we call this stopUpload or something that makes a bit more sense than breakout